deploy/src/main/deploy/helm/values.yaml

```diff
@@ -5,17 +5,31 @@
 # amqpMessagingNetworkDeployExample indicates whether the example AMQP Messaging Network
 # consisting of a single Dispatch Router and Artemis broker should be
 # deployed and used.
+# If this property is set to false, then the protocol adapters need to be
+# explicitly configured for connecting to an existing AMQP Messaging Network
+# using properties
+# - "adapters.amqpMessagingNetworkSpec"
+# - "adapters.commandAndControlSpec"
 amqpMessagingNetworkDeployExample: true
 # dispatchRouterImageName contains the name (including tag)
 # of the container image to use for the example AQMP Messaging Network
+# This property is only used if "amqpMessagingNetworkDeployExample" is set to true
 dispatchRouterImageName: ${dispatch-router.image.name}
 # artemisImageName contains the name (including tag) of the container
 # image to use for the example AMQP Messaging network
+# This property is only used if "amqpMessagingNetworkDeployExample" is set to true
 artemisImageName: ${artemis.image.name}
 
 
 # deviceRegistryDeployExample indicates whether the example Device Registry
 # should be deployed and used.
+# If this property is set to false, then the protocol adapters need to be
+# explicitly configured for connecting to the Tenant, Device Registration,
+# Credentials and Device Connection services using properties
+# - "adapters.tenantSpec"
+# - "adapters.deviceRegistrationSpec"
+# - "adapters.credentialsSpec"
+# - "adapters.deviceConnectionSpec"
 deviceRegistryDeployExample: true
```

Comment:
I might be missing it, but I don't see this value being use to actually decide wether to deploy the example registry or not?

Reply:
We use it to determine if the adapters should be configured to connect to the example registry but we are not using it (yet) to prevent the example registry from being deployed in the first place. I wonder how I could miss that ... thanks for pointing it out :+1: